packages/minuteman/extra/setup_iptables.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+# http://stackoverflow.com/questions/2870992/automatic-exit-from-bash-shell-script-on-error
+# Any subsequent(*) commands which fail will cause the shell script to exit immediately
+
+RULE="POSTROUTING -m ipvs --ipvs --vdir ORIGINAL --vmethod MASQ -m comment --comment Minuteman-IPVS-IPTables-masquerade-rule -j MASQUERADE"
+
+if ! iptables --wait -t nat -C ${RULE}; then
+  echo "Minuteman IPTables Rule not found: ${RULE}"
+  echo "Inserting Rule"
+  iptables --wait -t nat -I ${RULE} && echo "Rule successfully inserted"
+else
+  echo "Minuteman IPTables rule found"
```

Comment:
1. Customers don't have a rule with the same name already existing which causes our software to behave very oddly and be really hard to debug why it failed
2. The rule didn't get messed up via interactions with things like firewalld which reset all the iptables rules periodically
3. the rule didn't get messed up via interactions with docker bridge networking
4. the rule didn't get messed up via interactions with mesos overlay code

Ideally want to verify the rule / chain exists and is doing the right thing. And if it isn't say if it's a form we recognize (Either none or one of the previous versions) and update / fix it.